project/Version.scala

```diff
@@ -18,7 +18,11 @@ object Version {
   val postgres               = "42.6.0"
   val h2                     = "2.1.214"
   val apacheCommons          = "2.6"
-  val keycloak               = "25.0.3"
+  val keycloakAdapter        = "25.0.3"
+  val keycloakCore           = "25.0.6"
+//  val keycloakCore           = "26.1.4"
```

Comment:
seems the deployed instance needs to be updated accordingly to have the latest dep work.